vstools/utils/vs_proxy.py

```diff
@@ -169,10 +169,12 @@ def clear_cache() -> None:
         try:
             for output in get_outputs().values():
                 if isinstance(output, VideoOutputTuple):
-                    output.clip.get_frame(0)
+                    with output.clip.get_frame(0):
+                        pass
                     break
         except Exception:
-            core.std.BlankClip().get_frame(0)
+            with core.std.BlankClip().get_frame(0):
+                pass
```

Comment:
Not sure if applicable here. What even is the purpose of getting (and immediately discarding) a blank clip's first frame?

Reply:
Not sure. Figuring out why and whether there's a better method is out of the scope of this PR though imo

Reply:
This function modifies VapourSynth’s cache limit, so perhaps this is just an arbitrary VapourSynth call to trigger an actual cache update.